spec/support/shared_entity_specs.rb

```diff
@@ -133,3 +133,55 @@ def verify_signature(pubkey, signature, signed_string)
     end
   end
 end
+
+shared_examples "a JSON Entity" do
+  it "#to_json output matches JSON schema" do
+    json = described_class.new(data).to_json
+    expect(json.to_json).to match_json_schema(:entity_schema)
+  end
+
+  include_examples "common Entity JSON expectations" do
```

Comment:
Okay, and what do you think of `JSON is parsable with #from_json` below? Should I merge it with `a JSON Entity` as well?